src/model/documents/category.ts

```diff
@@ -0,0 +1,8 @@
+import { BaseModel } from '../base-model';
+import { CategoryType } from '../enums/category-type';
+
+export interface Category extends BaseModel {
```

Comment:
We can consider PTU rates in the future as a category prop.

Reply:
I believe it works in Poster like this - you add an article to the category and the PTU rate gets inherited, but you can override PTU rate for specific article/recipe if you wish.

Reply:
That's good idea, but probably for the future functionality after creating basic working application (MVP).